library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -212,6 +212,24 @@ class Client() {
         return ffiClient.signWithInstallationKey(message)
     }
 
+    fun verifySignature(message: String, signature: ByteArray): Boolean {
+        return try {
+            ffiClient.verifySignedWithInstallationKey(message, signature)
+            true
+        } catch (e: Exception) {
+            false
+        }
+    }
+
+    fun verifySignatureWithInstallationId(message: String, signature: ByteArray, installationId: String): Boolean {
```

Comment:
Sorry, I missed this when I approved. Can we rename this function to match with the ffi client?

Reply:
Or I can rename the fn on the client if it's confusing.

Reply:
I think it's confusing for integrators since it's not clear that the installationId is the public key. I think it's okay for the names between the SDKs and libxmtp to differ. This isn't the first time I've strayed from the ffi naming in benefits of clarity in the sdk.